src/index.js

```diff
@@ -304,5 +306,13 @@ Branch.prototype.creditHistory = function creditHistory() {
   return execute("getCreditHistory");
 };
 
+Branch.prototype.crossPlatformIds = function crossPlatformIds() {
```

Comment:
Don't think we need to expose cpid or latd.

Reply:
👍 wasn't sure, I'll remove them